src/app/views/scorecards/city_scorecard.js

```diff
@@ -2,15 +2,24 @@ define([
   'jquery',
   'underscore',
   'backbone',
+  // TODO do we need this old file for the city scorecard?
```

Comment:
I can remove this note if we want to leave as is, but as mentioned in the PR, I wonder if we should just remove all the city scorecard stuff. Otherwise, I could bring back the old version of this file until that decision gets made?

Reply:
I think it's fine to remove any and all of the old city scorecard stuff. These were never used as far as I understand.